rfc6962-bis.xml

```diff
@@ -1470,12 +1470,12 @@ but it is expected there will be a variety.
         </section>
       </section>
       <section title="TLS Feature Extension">
-	<t>
-	  If a certificate in a chain includes the <xref target="tls_transinfo_extension">transparency_info</xref> extension number in a <xref target="RFC7633">TLS Feature Extension</xref>, then CT compliance is required for all certificates which are directly or indirectly issued by that certificate.
-	</t>
-	<t>
-	  Clients MUST treat certificates which fail this requirement as non-compliant.
-	</t>
+        <t>
+          If any certificate in a chain includes the <xref target="tls_transinfo_extension">transparency_info</xref> TLS extension identifier in the <xref target="RFC7633">TLS Feature</xref> certificate extension, then CT compliance (using any of the mechanisms from <xref target="tls_servers"/>) is required.
+        </t>
+        <t>
+          Clients MUST treat certificates which fail this requirement as non-compliant.
```

Comment:
I've done that in https://github.com/robstradling/certificate-transparency-rfcs/commit/b067f25730fa2a7ee544eae7f684cc41b983d5f0